docs/getting-started/index.md

```diff
@@ -3,52 +3,55 @@ title: Getting started
 section: getting-started
 ---
 
-Developers should have some prior knowledge of the [Shopify app ecosystem](https://shopify.dev/concepts/apps). Currently, Shopify App CLI creates apps using either [Node.js](https://nodejs.org/) or [Ruby on Rails](https://rubyonrails.org/).
+Developers should have some prior knowledge of the [Shopify app ecosystem](https://shopify.dev/concepts/apps). Shopify App CLI creates apps using either [Node.js](https://nodejs.org/) or [Ruby on Rails](https://rubyonrails.org/).
 
-## Install
+## Requirements
 
-Shopify App CLI installs using a shell script. Download and run it in your terminal with one command:
+- [Ruby](https://www.ruby-lang.org) 2.5.1+ 
+- [Node.js](https://nodejs.org) 10.0.0+
+- [curl](https://curl.haxx.se) 7.0.0+
+- A [Shopify partner account](https://partners.shopify.com/signup)
+- A [Shopify development store](https://help.shopify.com/en/partners/dashboard/development-stores#create-a-development-store) to install and test apps
 
-### Mac OS and Ubuntu
-```console
-eval "$(curl -sS https://raw.githubusercontent.com/Shopify/shopify-app-cli/master/install.sh)"
-```
+### Windows requirements
 
-### Windows
-Install [Linux Subsystem for Windows](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and the [Ubuntu VM](https://www.microsoft.com/en-ca/p/ubuntu/9nblggh4msv6), then:
+You’ll need to install the following tools to use Shopify App CLI on Windows:
 
-```console
-eval "$(curl -sS https://raw.githubusercontent.com/Shopify/shopify-app-cli/master/install.sh)"
-```
+- [Linux Subsystem for Windows](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+- [Ubuntu VM](https://www.microsoft.com/en-ca/p/ubuntu/9nblggh4msv6)
 
-> NOTE: Installing the Shopify App CLI requires [curl](https://curl.haxx.se/). You can to see if it's on your system by running: `curl --version`
+## Install
 
-## Requirements to create a project
+Shopify App CLI installs using a [shell script](https://raw.githubusercontent.com/Shopify/shopify-app-cli/master/install.sh). Download and run it in your terminal with one command:
```

Comment:
I wonder about making the purpose of the link a bit more explicit, like "shell script (view source)"?

Reply:
Yeah, I'd be into that. I think it's just a good trust marker to encourage people to read over shell scripts before diving in